docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -6,9 +6,11 @@ This feature provides an early warning system for command and control beaconing
 [discrete]
 === Deploy the package
 
-To deploy the network beaconing framework in your environment using the https://github.com/elastic/detection-rules/releases/tag/ML-Beaconing-20211216-1[release package], follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+To deploy the network beaconing framework in your environment, follow https://docs.elastic.co/integrations/beaconing#installation[these steps].
```

Comment:
This path could use the [attribute](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L127) for integrations docs:
```suggestion
To deploy the network beaconing framework in your environment, follow {integrations-docs}/beaconing#installation[these steps].
```